src/components/Button/Button.component.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+
+import {Icon} from '@Components/Icon';
+
+import {ButtonProps} from './Button';
+import {StyledButtonContainer} from './Button.styles';
+
+/**
+ * Display the icon and text in button
+ */
```

Comment:
use single line comment

Reply:
The benefit of using multiline comments is that 
In vscode, wherever you use this component you'll be able to see what this component does by just hovering over the component. You won't have to go to the file where this component is written to check fo comments
![Screenshot from 2022-10-20 13-23-06](https://user-images.githubusercontent.com/102018792/196892228-36aa0972-3dc4-4f00-befe-8109eb57c6fc.png)